templates/hint.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function Hint (props) {
+  const {
+    _globals,
+    _hint
+  } = props;
+
+  return (
+    <div className={classes(["hint__inner"])}>
```

Comment:
```suggestion
    <div className="hint__inner">
```
`classes` function is for flattening, filtering and joining multiple classes array where some classes are falsey due to boolean flags. It lives here: https://github.com/adaptlearning/adapt-contrib-core/blob/de6fcd9c58ad7ed0d793920b048fcb213bdcc980/js/reactHelpers.js#L95-L101